src/renderer/components/attachment/Attachment.tsx

```diff
@@ -0,0 +1,83 @@
+import { ipcRenderer } from 'electron'
+import mimeTypes from 'mime-types'
+
+/* Section - Data Copied in part from Signal */
+// Supported media types in google chrome
+// See: https://en.wikipedia.org/wiki/Comparison_of_web_browsers#Image_format_support
+const SUPPORTED_IMAGE_MIME_TYPES = Object.freeze([
+  'image/bmp',
+  'image/gif',
+  'image/jpeg',
+  'image/jpg',
+  // No need to support SVG
```

Comment:
Just curious: do you know why there is no need to support SVG?

Reply:
I'm kinda curious too, this code was copied from signal so I don't know why. It was previously in the now deleted GoogleChrome.js

Reply:
Ok, let's leave it at that.